core/src/main/scala/org/genivi/sota/core/AutoInstallResource.scala

```diff
@@ -5,55 +5,27 @@
 package org.genivi.sota.core
 
 import akka.actor.ActorSystem
-import akka.http.scaladsl.server.AuthorizationFailedRejection
 import akka.http.scaladsl.server.Directives
 import akka.http.scaladsl.server.Directive1
 import akka.http.scaladsl.server.Route
 import akka.stream.ActorMaterializer
-
-import eu.timepit.refined.api.Refined
-
 import org.genivi.sota.common.DeviceRegistry
 import org.genivi.sota.core.db.AutoInstalls
-import org.genivi.sota.data.Namespace
-import org.genivi.sota.data.PackageId
-import org.genivi.sota.data.Uuid
-import org.genivi.sota.device_registry.common.{Errors => DeviceRegistryErrors}
+import org.genivi.sota.data.{DeviceDirectives, Namespace, PackageId, Uuid}
 import org.genivi.sota.http.AuthedNamespaceScope
 import org.genivi.sota.http.ErrorHandler
 import org.genivi.sota.http.Scopes
-import org.genivi.sota.http.UuidDirectives.extractUuid
 import org.genivi.sota.marshalling.CirceMarshallingSupport
-import org.genivi.sota.marshalling.RefinedMarshallingSupport._
 
 import scala.concurrent.ExecutionContext
-
 import slick.driver.MySQLDriver.api.Database
 
 class AutoInstallResource
-  (db: Database, deviceRegistry: DeviceRegistry, namespaceExtractor: Directive1[AuthedNamespaceScope])
-  (implicit system: ActorSystem, mat: ActorMaterializer, ec: ExecutionContext) extends Directives {
+  (db: Database, val deviceRegistry: DeviceRegistry, namespaceExtractor: Directive1[AuthedNamespaceScope])
```

Comment:
nah, if it doesn't compile just leave it in. I thought it might have been accidentally left in as it's the second variable which is marked as val. What happens when you move the val in front of db instead?